client/login/ssr.js

```diff
@@ -12,10 +14,14 @@ const VALID_QUERY_KEYS = [ 'client_id', 'signup_flow', 'redirect_to' ];
  * @param {Function} next     Next middleware in the running sequence
  */
 export function setShouldServerSideRenderLogin( context, next ) {
+	const isLocaleValidForSSR =
```

Comment:
It could be helpful to add a comment describing why we're doing this only for Mag-16 languages. It can be useful for anyone wondering why we're doing this for `/log-in` when for `/plugins` we directly redirect non-Mag-16 languages to the default locale.